cosmo_tester/test_suites/cluster/cfy_cluster_manager_shared.py

```diff
@@ -12,7 +12,7 @@
 BASE_VERSIONS = [
     version + '-ga'
     for version in SUPPORTED_RELEASES
-    if version not in ('master', '5.0.5')
+    if version not in ('master', '5.0.5', '5.1.0')
```

Comment:
The same change is needed in cosmo_tester/test_suites/general/cfy_manager_test.py I think. Maybe factor both out into constants.py as a SUPPORTED_FOR_RPM_UPGRADE?